### PR TITLE
chore: remove stale-issue-checker, add triage-bot workflow

### PR DESCRIPTION
**Pull request by mason-at-pieces**

### CODE REVIEW

File: .github/workflows/support-triage-bot.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  issues: write
+  contents: read
+
+jobs:
+  run-private-script:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v4
```
Comment (hra42) [inline .github/workflows/support-triage-bot.yml:17]: @mason-at-pieces you might wanna use v5 tag for this, v4 is over a year old:
https://github.com/actions/setup-python/releases

File: .github/workflows/support-triage-bot.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    - name: Setup SSH
+      run: |
+          mkdir -p ~/.ssh
+          echo "${{ secrets.ACTIONS_SSH_MM }}" > ~/.ssh/id_rsa
+          chmod 600 ~/.ssh/id_rsa
+          ssh-keyscan -t rsa github.com >> ~/.ssh/known_hosts
+
+    - name: Checkout Private Repository
+      uses: actions/checkout@v3
```
Comment (hra42) [inline .github/workflows/support-triage-bot.yml:29]: same here, checkout has a v4 tag available:
https://github.com/actions/checkout

File: .github/workflows/support-triage-bot.yml (hunk 1 above)
Comment (mason-at-pieces) [reply .github/workflows/support-triage-bot.yml]: updated, thanks!

File: .github/workflows/support-triage-bot.yml (hunk 2 above)
Comment (mason-at-pieces) [reply .github/workflows/support-triage-bot.yml]: updated, thanks!

**Review by mack-at-pieces — APPROVED** [review]

**Pull request merged**